### DISCUSSION TITLE
Quadra 700, Power Macintosh 7200

### DISCUSSION THREAD

**Discussion by AlexeyVolovik** [erichelgeson/BlueSCSI-v1#147] [Q&A]
Guys it is only 2 old macs I have. None of them recognizes BlueSCSI.

Log file looks good (I guess):
BlueSCSI <-> SD - https://github.com/erichelgeson/BlueSCSI
VERSION: 1.1-20220626-USB
DEBUG:0 SDFAT_FILE_TYPE:3
SdFat version: 2.1.2
Sd Format: exFAT
SPI speed: 50Mhz
SdFat Max FileName Length: 32
Initialized SD Card - lets go!
Sd MID:27 OID:PH
Sd Name:SD32G
Sd Date:1/2014
Sd Serial:3836256769
 - HD60_512 7.5.3.hda / 524288000bytes / 512000KiB / 500MiB
ID:LUN0:
 0:----:
 1:----:
 2:----:
 3:----:
 4:----:
 5:----:
 6: 512:
Finished initialization of SCSI Devices - Entering main loop.

No luck at all. I tried different images, different IDs, even tried to provide external powder. Nothing helps.

The only evidence that it is doing something is that when I put file name like HD00_512, mac stops to boot, so it is conflicting with hard drive. Other than that no evidences that all this BlueSCSI stuff is working anyhow.

Please help, I am very frustrated at the moment. So exciting YouTube videos, but real life looks confusing.

**Comment by erichelgeson** [comment]
First - please take a look at the [troubleshooting page ](https://github.com/erichelgeson/BlueSCSI/wiki/Troubleshooting)and confirm you've taken all those steps. It covers almost all use cases of issues.

After that - to help we'll need more info:
* What model/version is this?
* Is this a kit you built or did you buy it form someone on bluescsi.com?
* Where did you get/how did you create the image?
* Do you see the image in disk utility or any formatting utility?
* Is termination enabled if it is the first or last device in the scsi chain?

Again the [troubleshooting doc](https://github.com/erichelgeson/BlueSCSI/wiki/Troubleshooting)goes through all this too.

**Reply by AlexeyVolovik** [reply]
Yes, I have checked troubleshooting page and tried all the things are there. As well I red all the forum brunches that looked more or less helpful.

Here is more info:

It is BlueSCSI DB25 1.1-a. Firmware 20220626-STM32F1-USB.

I bought it from Onegeekarmy fully assembled.

I tried few different images. One I created with Disk Jockey, another one I took from https://mega.nz/folder/8hA3AQCJ#pWUq92L70yDXlogy9lk5Dg and as well I tried blank 2Gb image from https://github.com/erichelgeson/BlueSCSI/blob/3b17788cc677b7b089875786be94f929d4913bce/docs/MacHD.zip .

On SheepShaver side images look fine. SD Card looks fine as well, I even have a Log file written on it. On classic macs side I do not have any utilities. I am new to all this classic macs thing, bought Quadra 700 to make it my daily retro computer and 7200 for spare parts.

I guess as far as it is external BlueSCSI it is no options about termination, it is always on.

**Reply by erichelgeson** [reply]
Just to verify - you've done a full overwrite format with "SD Formatter Utility" and tried a different SD card correct? Also those blanks (MacHD.zip) were deleted from the repo so please dont use them, they have LIDO drivers and will not work properly.

The disk jockey ones should work if you're using the latest v2.0.1.

It may be silly but has happened - could you make sure you've pushed the db25 connection all the way in? If it's not seated properly then it could might not be making secure connection.

It sounds like the Q700 has a internal drive with an OS on it correct? Try to make a fresh DJ image and put it on the card - boot into the old hdd and look for Disk Utility or Apple SC HD Setup.

Lastly could you describe what you see on the LED's of the bluescsi?

**Reply by AlexeyVolovik** [reply]
I have some results for you. Mostly negative, but something started to work.

Originally I used 32Gb SD card. When I changed SD card to another one 16Gb. 7200 started to see 2 images I have created, one for System, one for Games. Image for System even listed files on it. Images were from blank images archive originally, one that you did not recommend.

Problem that i have real HDD only on 7200, Quadra do not have HDD. 7200 do not have any disk utilities.

When I put same SD card into Quadra it started to boot Mac OS, but stopped on some extension. I decided to play with Disk Jockey and Mac OS versions. I have Disk Jokey 2.1. Literally zero of images I have created with it worked.

So I dropped Quadra and decided to switch to 7200, because it was positive results there. I decided to copy Warcraft on BlueSCSI and try to play it. I didn't succeeded, folder became corrupted when I opened it on 7200. More I experimented worse it went. I finished when BuleSCSI images stopped to be visible even on 7200 at all.

My best result I have copied a file from 7200 to BuleSCSI drive and then I opened same fine in SheepShaver on my modern Mac.

Wuf, I do not know why I am so special with all this complexity, but for now it all looks very broken.

Leds are fine. I have one red all the time and green when it is reading or writing data.

Do not know what to do really. Maybe I should try one another SD card? It all so strange.

**Reply by erichelgeson** [reply]
Can you verify you have formatted the SD card with SD Formatting Utility linked in the Usage page with overwrite selected? And just use ONE image at a time, ensuring it's one from the latest disk jockey, and never use those MacHD.zip images.

Second if you have a multi meter please test the voltage on pin 25 of the db25 connector for 5v - some powermacs dont supply enough 5v voltage on their 5v rail and can cause improper termination if not powered externally, which may lead to erratic behavior. 

Just slow down and try the simplest thing first - one fresh image from DJ, on a freshly formatted card that has been formatted with the SD Formatting utility - powered via USB.

**Reply by AlexeyVolovik** [reply]
Yes, I am using SD Card Formatter.

I do not have multimeter, but I have MacBook power supply, that I used as an external power source. External power supply do not change situation anyhow, so I guess it is all fine with 5v.

I found one strange thing. More I experiment, worse it work. If I leave BlueSCSI for like a half of the day, it starts to show drives. It is different weird things there, but it starts to show drives. Doing experiments like an hour I became to situation when none images are working. Literally, even copies of images that were worked before.

I waited like 3 hours. Turned 7200 on again. File name was HD60_512.hda, so it is even tried to boot from BlueSCSI. Unsuccessfully. It showed boot screen, started to load something and freeze. Now it boots to internal hard drive. BlueSCSI do not work again completely. Buy the way similar behavior was for Q700.

I guess all this not because of images I use or macs I use. I guess it is something technical. Honestly hard to imagine what can it be, basically it is just STM32 soldered to the connector.

OK, I have a good news finally. I took 64Gb SD card from my GoPro to check. I all started to work just fine. I booted up 7200 from System 7.5.3 image. I booted up Quadra 700 from System 7.5.3 image. I all looks good.

It appears that BlueSCSI is quite picky on SD Cards.

**Reply by erichelgeson** [reply]
Glad you got it working!
